src/helpers/Url/Url.js

```diff
@@ -94,14 +94,16 @@ export function getView(url) {
  */
 export function flattenToAppURL(url) {
   const { settings } = config;
+
   return (
     url &&
-    url
+    `${settings.prefixPath}${url
```

Comment:
I have a problem with listing blocks (and maybe in other parts).

in [universalLink](https://github.com/plone/volto/blob/master/src/components/manage/UniversalLink/UniversalLink.jsx) the url will be "flattened" several times and with your changes, i have links inside listing blocks like "/my-domain/my-domain/example-page".

Probably in universalLink code we should avoid multple flattenToAppURL calls, but maybe also in that function we should avoid to append the same string several times.

Reply:
I am more inclined towards the latest solution. This function should be idempotente imho, so it should not append the prefix every time it's called.
I wouldn't check if the url already starts with the prefixPath. Maybe we should only add the prefixPath if the url starts with ``http``?